src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -464,161 +462,142 @@ const addressUISchema = (addressName, title) => {
   };
 };
 
-const addressSchema = (isRequired = false) => {
-  return {
-    type: 'object',
-    required: isRequired ? ['country', 'addressLine1'] : [],
-    properties: {
-      type: {
-        type: 'string',
-        'enum': ['MILITARY', 'DOMESTIC', 'INTERNATIONAL']
-      },
-      country: {
-        type: 'string',
-        'enum': countries
-      },
-      state: {
-        type: 'string',
-        'enum': states
-      },
-      addressLine1: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
-      },
-      addressLine2: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
-      },
-      addressLine3: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
-      },
-      city: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.#]([a-zA-Z0-9-'.# ])?)+$"
-      },
-      zipCode: {
-        type: 'string'
+const addressSchema = {
```

Comment:
Is the idea to use the PCIU address schema for this eventually?

Reply:
I think so, yes.